ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -92,12 +70,18 @@ std::unique_ptr<Process> createHTProcess(
         "density_solid", parameters, 1);
     DBUG("Use \'%s\' as density_solid parameter.", density_solid.name.c_str());
 
+    //! \ogs_file_param_special{prj__processes__process__HT__fluid__density}
+    auto const& fluid_density_conf = fluid_config.getConfigSubtree("density");
```

Comment:
Fluid density, viscosity, heat capacity and thermal conductivity can be represented by FluidProperties and its instance can be created by calling created by createFluidProperties(BaseLib::ConfigTree const& config) here. (see  LiquidFlowMaterialProperties.h/cpp).